dataplane/windows/policysets/policysets.go

```diff
@@ -559,3 +564,11 @@ func filterNets(mixedCIDRs []string, ipVersion uint8) (filtered []string, filter
 	}
 	return
 }
+
+// AppendPrefixToPolicyNames appends a prefix to each policy name
+func AppendPrefixToPolicyNames(prefix string, policyNames []string) (appendedNames []string) {
```

Comment:
Nit: I think you're "prepending", not "appending"

Since this function just takes some strings and returns updated strings, I'd be tempted to move it to the package where it's used and make it more generic (and a bit shorter in name) `prependAll(prefix string, in []strings) []string`

Reply:
Sounds good, i'll make this change.